stockflux-components/src/components/ScrollWrapperY/ScrollWrapperY.js

```diff
@@ -0,0 +1,37 @@
+import React, { useRef, useEffect } from "react";
+import "./ScrollWrapperY.css";
+
+const ScrollWrapperY = props => {
+  const scrollWrapperRef = useRef(null);
+
+  useEffect(() => {
+    checkIfOverflowing();
+    window.addEventListener("resize", checkIfOverflowing);
+    return () => {
+      window.removeEventListener("resize", checkIfOverflowing);
+    };
+  }, []);
+
+  useEffect(() => {
+    checkIfOverflowing();
+  }, [props.contentChanged]);
+
+  const checkIfOverflowing = () => {
+    if (scrollWrapperRef.current)
+      if (
```

Comment:
Could shorten this to
`
scrollWrapperRef.current.classList.toggle('scroll-padding-right', scrollWrapperRef.current.scrollHeight > scrollWrapperRef.current.clientHeight);
`

Reply:
That's neat! Thanks - > 08cff915e0c7c162c3c71c72e9adb0e02ba1de6c